src/server/reset_password.rs

```diff
@@ -10,7 +10,7 @@ use edgedb_client::credentials::Credentials;
 use fn_error_context::context;
 use rand::{Rng, SeedableRng};
 
-use crate::credentials;
+use crate::credentials as creds_crate;
```

Comment:
Kinda weird name I would read it as "credentials crate", so it should be a separate crate rather than a module within the current one.

Also we `use crate::credentials;` everywhere I think and I don't see a name conflict here.

Reply:
Yeah you're right - fixed!